packages/create-instance/create-scoped-slots.js

```diff
@@ -36,25 +36,12 @@ function getVueTemplateCompilerHelpers (): { [name: string]: Function } {
 }
 
 function validateEnvironment (): void {
-  if (window.navigator.userAgent.match(/PhantomJS/i)) {
-    throwError(
-      `the scopedSlots option does not support PhantomJS. ` +
-        `Please use Puppeteer, or pass a component.`
-    )
-  }
   if (vueVersion < 2.5) {
```

Comment:
If template tag in scopedSlots mounting options works normally,
I think this limitation is not necessary.